playbooks/rpm_packaging.yml

```diff
@@ -0,0 +1,38 @@
+---
+- hosts: all
+  tasks:
+    - name: Install dependencies for packageing tooling
+      yum:
+        name: "{{ package_list }}"
+        state: present
+      vars:
+        package_list:
+          - git-core
+          - git-annex
+          - rubygem-nokogiri
+          - rpmspectool
+          - rubygem-gem2rpm
+          - make
+          - gcc
+          - redhat-rpm-config
+          - ruby-devel
+          - curl-devel
+          - gem
+          - python3-click
+          - python3-ruamel-yaml
+          - crudini
+      become: true
+
+    - name: Install gem-compare
```

Comment:
I use https://copr.fedorainfracloud.org/coprs/ekohl/gem-compare/ and saves the need for having `gem` and all `-devel` packages.